addons/stock_account/views/product_views.xml

```diff
@@ -33,6 +33,7 @@
             <field name="arch" type="xml">
                 <field name="property_account_expense_categ_id" position="after">
                     <field name="property_stock_valuation_account_id" string="Stock Account" options="{'no_create': True}"/>
+                    <field name="account_stock_variation_id" string="Stock Variation" options="{'no_create': True}"/>
```

Comment:
Should it be `readonly`? :thinking: 
Changing it in the product category form doesn't update it anyway, even if it doesn't complain while saving.